### ISSUE TITLE
(aws_ecs_patterns.ApplicationLoadBalancedServiceBase): daemon scheduling strategy

### ISSUE THREAD

**Issue by jrobbins-LiveData** [aws/aws-cdk#14107]
Testing `ApplicationLoadBalancedEc2Service`  in CDK version 1.97.0.

The [documentation](https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_ecs_patterns/ApplicationLoadBalancedEc2Service.html?highlight=daemon) refers to `daemon` twice, but doesn't seem to say how to set the scheduling strategy to `daemon`.

The [Ec2Service](https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_ecs/Ec2Service.html) construct has a `daemon` parameter.  So I'm wondering if this is an oversight in the definition of `ApplicationLoadBalancedEc2Service`?

**Comment by peterwoodworth** [comment]
@SoManyHs mind giving your take on this one?

**Comment by SoManyHs** [comment]
Yep -- this is a gap as far as properties on the Service being exposed on the higher-level pattern. We are currently in the process of re-designing some of our patterns (See: https://github.com/aws/aws-cdk/issues/14625), which will hopefully make them more configurable!
